src/sagas/wallet.js

```diff
@@ -380,41 +380,6 @@ export function* fetchTokensMetadata(tokens) {
   yield put(tokenMetadataUpdated(tokenMetadatas, errors));
 }
 
-// This will create a channel to listen for featureFlag updates
-export function* listenForFeatureFlags(featureFlags) {
-  const channel = eventChannel((emitter) => {
-    const listener = (state) => emitter(state);
-    featureFlags.on(FeatureFlagEvents.WALLET_SERVICE_ENABLED, (state) => {
-      emitter(state);
-    });
-    featureFlags.on(FeatureFlagEvents.ATOMIC_SWAP_ENABLED, (state) => {
-      emitter(state);
-    });
-
-    // Cleanup when the channel is closed
-    return () => {
-      featureFlags.removeListener(FeatureFlagEvents.WALLET_SERVICE_ENABLED, listener);
-      featureFlags.removeListener(FeatureFlagEvents.ATOMIC_SWAP_ENABLED, listener);
-    };
-  });
-
-  try {
-    while (true) {
-      const newUseWalletService = yield take(channel);
-      const oldUseWalletService = yield select((state) => state.useWalletService);
-
-      if (oldUseWalletService && oldUseWalletService !== newUseWalletService) {
```

Comment:
Are you doing this check somewhere? I didn't see it. In the mobile PR it's in this same file